src/gluonts/shell/__main__.py

```diff
@@ -154,9 +154,11 @@ def train_command(data_path: str, forecaster: Optional[str]) -> None:
     from gluonts.shell import train
 
     logger.info("Run 'train' command")
-    env = TrainEnv(Path(data_path))
+    data_path = Path(data_path)
+    failure_path = data_path / "output" / "failure"
```

Comment:
I think it's better to instantiate a `TrainPaths`-object here and pass it to the `TrainEnv`.

Reply:
changed it, please review it